lib/internal/main/print_help.js

```diff
@@ -164,7 +164,8 @@ function print(stream) {
   options.set('--', { helpText: 'indicate the end of node options' });
   stream.write(
     'Usage: node [options] [ -e script | script.js | - ] [arguments]\n' +
-    '       node inspect script.js [arguments]\n\n' +
+    '       node inspect script.js [arguments]\n' +
+    '       node inspect 192.0.0.1:9229\n\n' +
```

Comment:
```suggestion
    '       node inspect <host>:<port>\n\n' +
```
?

Reply:
or maybe:
```suggestion
    '       node inspect [script.js | -e "script" | <host>:<port>] …\n\n' +
```

to match https://github.com/nodejs/node/blob/master/doc/api/cli.md#synopsis?